en/getting_started/px4_basic_concepts.md

```diff
@@ -129,24 +129,20 @@ PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_rep
 A number of recommended cards are listed in: [Developer Guide > Logging](http://dev.px4.io/en/log/logging.html#sd-cards)
 
 
-## Disarmed/Pre-armed/Armed {#arming}
+## Arming and Disarming {#arming}
 
 Vehicles may have moving parts, some of which are potentially dangerous when powered (in particular motors and propellers)!
 
-To reduce the chance of accidents, PX4 has explicit state(s) for powering the vehicle components:
-- **Disarmed:** There is no power to motors or actuators.
-- **Pre-armed:** Actuators and other non-dangerous electronics are powered.
-  - In this state you can move ailerons, flaps etc, but motors/propellers are locked.
-- **Armed:** Vehicle is fully powered, including motors/propellers.
+To reduce the chance of accidents:
+- PX4 vehicles are *disarmed* (unpowered) when not in use, and must be explicitly *armed* before taking off.
+- Some vehicles additionally require a [safety switch](../getting_started/px4_basic_concepts.md#safety_switch) be disengaged before arming can be attempted.
```

Comment:
```suggestion
- Some vehicles additionally require a [safety switch](../getting_started/px4_basic_concepts.md#safety_switch) be disengaged before arming can succeed.
```
Not sure here, just should say you can attempt but it will not arm since it's part of the checks.

Reply:
Yes, this is a fine point, but a good one. I will just accept your suggestion.